### PR TITLE
Add compressibility quantity

### PR DESCRIPTION
**Pull request by tmilnthorp**
Closes #1032

### CODE REVIEW

File: Common/UnitDefinitions/Compressibility.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "XmlDoc": "Compressibility is the measure of the relative volume change of a fluid or solid in response to pressure changes.",
+  "BaseDimensions": {
+    "L": 1,
+    "M": -1,
+    "T": 2
+  },
+  "Units": [
+    {
+      "SingularName": "InversePascal",
+      "PluralName": "InversePascals",
```
Comment (angularsen) [inline Common/UnitDefinitions/Compressibility.json:13]: Would any other units be useful to add also @sveipa? I don't know this domain.

https://petroleumoffice.com/unitconverter/compressibility/
https://www.unitsconverters.com/en/Compressibility-Conversions/Measurement-1170
https://www.calculand.com/unit-converter/?gruppe=Compressibility&einheit=Second+per+poise+%5Bs%2FP%5D

**Review by angularsen — APPROVED** [review]

File: Common/UnitDefinitions/Compressibility.json (hunk 1 above)
Comment (sveipa) [reply Common/UnitDefinitions/Compressibility.json:13]: We did a scan of our legacy code here and these are the important units + scales for compressibilty:
Pa-1 - 1/Pascal

kPa-1 - 1/kiloPascal

MPa-1 - 1/MegaPascal

atm-1 - 1/atmosphere

mbar-1 - 1/millibar

bar-1 - 1/bar

psi-1 - 1/pounds per square inch

File: UnitsNet.Tests/CustomCode/CompressibilityTests.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    public class CompressibilityTests : CompressibilityTestsBase
+    {
+        // Override properties in base class here
+
+        protected override double InversePascalsInOneInversePascal => 1;
+        protected override double InverseMillibarsInOneInversePascal => 1e-2;
+        protected override double InverseKilopascalsInOneInversePascal => 1e-3;
+        protected override double InverseMegapascalsInOneInversePascal => 1e-6;
+        protected override double InverseBarsInOneInversePascal => 1e-5;
+        protected override double InversePoundForcePerSquareInchsInOneInversePascal => 1.450377377302092151542e-4;
```
Comment (angularsen) [inline UnitsNet.Tests/CustomCode/CompressibilityTests.cs:33]: Typo `Inchs`

**Review by angularsen — CHANGES_REQUESTED** [review]
Found a minor typo, other than that you can go ahead and merge this

File: UnitsNet.Tests/CustomCode/CompressibilityTests.cs (hunk 2 above)
Comment (tmilnthorp) [reply UnitsNet.Tests/CustomCode/CompressibilityTests.cs]: > Typo `Inchs`

Ah indeed. It should be ```InversePoundsForcePerSquareInch``` anyways. Fixed!

**Comment by angularsen** [comment]
Are you sure it's plural pounds in InversePoundsForceInches?
In Torque, we used PoundForceInches.
https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/Torque.json#L67-L79

I see both forms used, but I seem to find more singular form than plural form.
https://en.wikipedia.org/wiki/Pound_(force)

**Comment by tmilnthorp** [comment]
> Are you sure it's plural pounds in InversePoundsForceInches? In Torque, we used PoundForceInches. https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/Torque.json#L67-L79
> 
> I see both forms used, but I seem to find more singular form than plural form. https://en.wikipedia.org/wiki/Pound_(force)

This is more associated with Pressure than Torque. That said, I'd love for @sveipa to clarify

**Comment by sveipa** [comment]
> > Are you sure it's plural pounds in InversePoundsForceInches? In Torque, we used PoundForceInches. https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/Torque.json#L67-L79
> > I see both forms used, but I seem to find more singular form than plural form. https://en.wikipedia.org/wiki/Pound_(force)
> 
> This is more associated with Pressure than Torque. That said, I'd love for @sveipa to clarify

I've run the singular vs plural case internally here without being able to conclude what's the right naming convention unfortunately. Sorry for not being able to help on this matter.

**Comment by angularsen** [comment]
If there is no clear cut convention, I vote we use singular form `InversePoundForceInches` since it is consistent with `PoundForceInches`. I don't have strong feelings either way, but I prefer consistency.

**Comment by sveipa** [comment]
Can this PR be merged please? :-)

**Comment by angularsen** [comment]
I checked the code base and we have 40/60 mix of PoundForce vs PoundsForce, so I guess there is no consistency to begin with. Keeping it PoundsForce then.

**Comment by angularsen** [comment]
Sorry for the slow turnaround @sveipa @tmilnthorp , nuget is on the way out.

[Release UnitsNet/4.129.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.129.0)

**Comment by sveipa** [comment]
Great, thanks @angularsen :-)

**Pull request merged**